turtlebot1:
  start_x: -2.485178409362728
  start_y: -1.0597343531451797
  start_yaw: 1.323989740484531
  goal_x: -3.929818037373357
  goal_y: 1.531461510195351
  goal_yaw: 4.587417792804202
turtlebot2:
  start_x: -3.918271744368877
  start_y: 1.7824721249174171
  start_yaw: 1.7051770229026246
  goal_x: -0.8814372298617839
  goal_y: -1.9570765390097073
  goal_yaw: 2.10985631177184
